lib/model/model.go

```diff
@@ -1602,6 +1602,7 @@ func (m *Model) internalScanFolderSubdirs(folder string, subs []string) error {
 		m.updateLocalsFromScanning(folder, batch)
 	}
 
+	m.folderStatRef(folder).ScanCompleted()
```

Comment:
So I wonder if we should update this if subs is not nil (for example inotify asking us to scan a specific file). Open to suggestions/discussions.

@uok @Zillode @canton7 @Nutomic @calmh